user_docs/en/changes.md

```diff
@@ -58,6 +59,10 @@ Please refer to [the developer guide](https://www.nvaccess.org/files/nvda/docume
 * It is now possible to redirect objects retrieved from on-screen coordinates, by using the `NVDAObject.objectFromPointRedirect` method. (#16788, @Emil-18)
 * Running SCons with the parameter `--all-cores` will automatically pick the maximum number of available CPU cores. (#16943, #16868, @LeonarddeR)
 * Developer info now includes information on app architecture (such as AMD64) for the navigator object. (#16488, @josephsl)
+* In the gecko_ia2 virtual buffer backend, the accValue is exposed for links (#141, @LeonarddeR).
```

Comment:
Thanks for mentioning me here, but I don't think this really needs a changelog entry.
```suggestion
```